framework/tests/annotationclassloader/Makefile

```diff
@@ -1,21 +1,23 @@
 # this makefile contains the commands to test the AnnotationClassLoader
 # it uses the aliasing checker as it is one of the framework checkers that has a qual directory with qualifiers that must be loaded using the loader
 
-JAVAOPTS=-source 7 -target 7
-
 # when called by the ant tests, JAVAC is defined as $JAVA_HOME/bin/javac, which could be jdk8 or jdk7
-# when executed as a demo, we'll use openjdk 8
-ifndef JAVAC
-	JAVAC?=/usr/lib/jvm/java-8-openjdk-amd64/bin/javac
-endif
+# when executed as a demo, we'll use openjdk 8 located in the following directory (change to your setup):
+# ?= sets JAVAC only if JAVAC is undefined
```

Comment:
I would remove the explanation of ?=. It's used in many other scripts, where it's not documented.